redshift_sqlalchemy/dialect.py

```diff
@@ -249,6 +250,20 @@ def _get_column_info(self, *args, **kwargs):
 
         return column_info
 
+    def create_connect_args(self, *args, **kwargs):
+        default_args = {
+            'sslmode': 'verify-full',
```

Comment:
It looks like the redshift cluster is sending a valid ServerCertificate (when validated with redshift-ssl-ca-cert.pem)  with the correct common name: "redshift-sqlalchemy-test.cforsfjmjsja.us-west-2.redshift.amazonaws.com" see https://gist.github.com/graingert/3a46c493520db7caa460#file-redshift-tls-server-hello-txt-L188